tests/admin_widgets/tests.py

```diff
@@ -1781,6 +1818,63 @@ def test_ForeignKey_using_to_field(self):
         self.assertEqual(profiles[0].user.username, username_value)
 
 
+class RelatedFieldWidgetSeleniumPrimaryKeyTests(AdminWidgetSeleniumTestCase):
+    def test_create_houses_and_room(self):
+        from selenium.webdriver.common.by import By
+        from selenium.webdriver.support.select import Select
+
+        self.admin_login(username="super", password="secret", login_url="/")
+
+        # Create a new House with a PK that needs quoting
+        house = House.objects.create(name="_40")
+        house_name = str(house.name)
+        self.selenium.get(
+            self.live_server_url + reverse("admin:admin_widgets_house_add")
+        )
+
+        self.selenium.find_element(By.ID, "id_name").send_keys(house_name)
+        self.selenium.find_element(By.NAME, "_continue").click()
+
+        # Check that the new House was created and listed in the change form
```

Comment:
To add more explanation, comments should state the expected behavior and omit prefixes like "Check that" since all tests check things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Thank you, I will work on it.